### ISSUE TITLE
Loading NBT data

### ISSUE THREAD

**Issue by allanf181** [IrisShaders/Iris#1632]
### Minecraft Version

1.18.2

### Iris Version

iris-mc1.18.2-1.3.0.jar

### Sodium Version

sodium-fabric-mc1.18.2-0.4.1+build.15.jar

### Operating System

Windows 10

### What is your GPU?

Nvidia GeForce RTX 2060

### Java Version

Java 17

### Reproduction Steps

1. Open a world or join in multiplayer
2. NBT Loading error

### Crash Report file and latest.log

```shell
There was an error creating your issue: body is too long, body is too long (maximum is 65536 characters).

https://paste.cflookup.com/BitRmXuHcr.yaml
```

### Additional context

![image](https://user-images.githubusercontent.com/14101716/189496832-77a4ea5e-9653-44e3-83f9-382d056de9c0.png)
It's not a crash but it prevents the game from working.
I know that the log may not make sense, but it only happens with Iris 1.3.0, in previous versions or without Iris the game works normally.

**Comment by IMS212** [comment]
Already fixed in main branch; Mod is using invalid characters and assuming an exception.

**Comment by allanf181** [comment]
![image](https://user-images.githubusercontent.com/14101716/189497897-3b2ebc60-61f8-406c-bb43-d781d0af96fe.png)
Using Iris 1.3.1

**Comment by allanf181** [comment]
@IMS212 I compiled a version without path validation, NBT error continues, is not the same error.

**Comment by IMS212** [comment]
We are aware, and are discussing on our Discord. Please do not provide updates here, as we are already aware of the core issue.

**Comment by IMS212** [comment]
Should be fixed with 1.3.1.
